ci_config.json

```diff
@@ -71,6 +71,20 @@
       "windows": false
     }
   },
+  "glfw": {
```

Comment:
The CI should enable any build options that are false by default.

Reply:
I don't think this got implemented, by the way?

Reply:
Oh! Let me fix that.